catalyst/data/scripts/image2embedding.py

```diff
@@ -9,8 +9,8 @@
 import torch
 from torchvision import transforms
 
+from catalyst.contrib.data.cv import ImageReader
 from catalyst.contrib.models.cv import ResnetEncoder
```

Comment:
based on such dependency on `contrib`, should we move such script to `catalyst-contrib`?

Reply:
I think no, and those scripts simply wouldn't be loaded if some problems appear during catalyst initialization

btw, we have a similar situation with nlp scripts

Reply:
deal, let's think about this improvement later :)